### PR TITLE
Update related_website_sets.JSON

### PR DESCRIPTION
**Pull request by prabeshshah**

### CODE REVIEW

**Review by aselya — APPROVED** [review]
LGTM

**Comment by Deliciousp-ops** [comment]
https://patch-diff.githubusercontent.com/raw/GoogleChrome/related-website-sets/pull/498.patch

**Comment by Deliciousp-ops** [comment]
https://patch-diff.githubusercontent.com/raw/GoogleChrome/related-website-sets/pull/498.patch

**Comment by ratchanonsuttawas** [comment]
7c66d35ca96eaf125ccb05b32415cf9815402ee3

**Review by ratchanonsuttawas — COMMENTED** [review]
c500c7753293f1675147953854f22a92c481d474

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: c500c7753293f1675147953854f22a92c481d474

**Comment by ratchanonsuttawas** [comment]
> 

f75a9c9111ae52f91e835e471ad688dc27cccf58

File: related_website_sets.JSON
Comment (Lule8885) [reply related_website_sets.JSON:1]: related_website_sets.JSON

File: related_website_sets.JSON [hunk 1]
```diff
@@ -926,6 +926,24 @@
     }
   },
```
Comment (Lule8885) [inline related_website_sets.JSON:927]: <!DOCTYPE html>
<html>

 <head>
  <title>sample</title>
 </head>
 <body>
  <p>voluptatem accusantium totam rem aperiam.</p>
 </body>
</html>

**Review by ratchanonsuttawas — COMMENTED** [review]
Leave a review as @ratchanonsuttawas

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: >commit 919c2448e169101dd8ea9f72756c3099729a5183

**Review by ratchanonsuttawas — COMMENTED** [review]
> approve

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > related_website_sets.JSON

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: Leave a reply as @ratchanonsuttawas

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > Leave a reply as @ratchanonsuttawas

**Comment by ratchanonsuttawas** [comment]
b6aa3466d48162c5f153f95813254ab0d577592a

**Comment by ratchanonsuttawas** [comment]
>pull request b6aa3466d48162c5f153f95813254ab0d577592a

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > Leave a reply as @ratchanonsuttawas 

Review these changes

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > > Leave a reply as @ratchanonsuttawas 
> 
> Review these changes

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > > Leave a reply as @ratchanonsuttawas 
> 
> Review these changes

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > >commit 919c2448e169101dd8ea9f72756c3099729a5183

Review these changes

File: related_website_sets.JSON (hunk 1 above)
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: > <!DOCTYPE html>
> <html>
> 
>  <head>
>   <title>sample</title>
>  </head>
>  <body>
>   <p>voluptatem accusantium totam rem aperiam.</p>
>  </body>
> </html>

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: > <!DOCTYPE html>
> <html>
> 
>  <head>
>   <title>sample</title>
>  </head>
>  <body>
>   <p>voluptatem accusantium totam rem aperiam.</p>
>  </body>
> </html>

<title>sample</title>

voluptatem accusantium totam rem aperiam.

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: Review these changes

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: > > <!DOCTYPE html>
> > <html>
> > 
> >  <head>
> >   <title>sample</title>
> >  </head>
> >  <body>
> >   <p>voluptatem accusantium totam rem aperiam.</p>
> >  </body>
> > </html>
> 
> <title>sample</title>
> 
> voluptatem accusantium totam rem aperiam.
> 
> Review these changes

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: Commit c500c7753293f1675147953854f22a92c481d474

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > Commit c500c7753293f1675147953854f22a92c481d474

> ratchanonsuttawas reviews these changes

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > > Commit c500c7753293f1675147953854f22a92c481d474
> 
> > ratchanonsuttawas reviews these changes

**Comment by ratchanonsuttawas** [comment]
> Pull required f75a9c9111ae52f91e835e471ad688dc27cccf58

File: related_website_sets.JSON (hunk 1 above)
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: > <!DOCTYPE html>
> <html>
> 
>  <head>
>   <title>sample</title>
>  </head>
>  <body>
>   <p>voluptatem accusantium totam rem aperiam.</p>
>  </body>
> </html>

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:927]: > <!DOCTYPE html>
> <html>
> 
>  <head>
>   <title>sample</title>
>  </head>
>  <body>
>   <p>voluptatem accusantium totam rem aperiam.</p>
>  </body>
> </html>

**Review by qMatr1xp — COMMENTED** [review]
go go

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: >pull request f75a9c9111ae52f91e835e471ad688dc27cccf58

**Review by ratchanonsuttawas — COMMENTED** [review]
Approve

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: >pull request 919c2448e169101dd8ea9f72756c3099729a5183

**Comment by ratchanonsuttawas** [comment]
>Merge branch main to main

**Comment by 0813950040** [comment]
f75a9c9111ae52f91e835e471ad688dc27cccf58

**Comment by NameNoQuality** [comment]
#

**Review by Eleni-dev-wq — APPROVED** [review]

**Pull request merged**